synapse/config/key.py

```diff
@@ -210,16 +219,34 @@ def generate_config_section(
         #
         #trusted_key_servers:
         #  - server_name: "matrix.org"
+        #
+
+        # The additional signing keys to use when acting as a trusted key server, on
+        # top of the normal signing keys.
```

Comment:
I kinda feel like that, if this option is specified at all, it should specify all of the keys (so you'd have to specify the normal signing_key again)